ui/angular.json

```diff
@@ -33,7 +33,8 @@
               "src/styles.scss"
             ],
             "scripts": [
-              "node_modules/@webcomponents/custom-elements/custom-elements.min.js",
+              "node_modules/@webcomponents/webcomponentsjs/custom-elements-es5-adapter.js",
+              "node_modules/@webcomponents/webcomponentsjs/webcomponents-bundle.js",
```

Comment:
What do you need these for?

Reply:
It has been added with a clarity upgrade. 
Step 3 in https://clarity.design/documentation/get-started